unittest/python/crocoddyl/robots.py

```diff
@@ -1,34 +1,17 @@
 import numpy as np
```

Comment:
Can we make a list of features + issues of what functionality of Python isnt yet available in C++

Reply:
There is some issues about it. However, I agree that we need to update and to create a single issue that enlisted. I will do it soon.

Reply:
I went for the easiest solution working for me. I agree we should completely remove robots.py, but it would require extra work for the things relying on it to work, even if they are meant to disappear soon, so I don't think it is worth it.

Reply:
Let's keep it like this. Any action is just wasting our time